app/views/accounts/index.html.erb

```diff
@@ -129,5 +129,8 @@
         <% end %>
       </tbody>
     </table>
+
+    <%= paginate @users, theme: 'twitter-bootstrap-4' %>
```

Comment:
after staging deploy it turned out that `.page(params[:page])` is missing for `AccountsController`. i will hotfix it right away @tonekk

Reply:
see #279